kolibri/plugins/learn/assets/src/vue/breadcrumbs/index.vue

```diff
@@ -11,7 +11,7 @@
           <breadcrumb :showarrow='false' :linkobject="exploreRootLink" :text="$tr('explore')"></breadcrumb>
         </span>
         <span class="portrait">
-          <breadcrumb :linkobject="parentExploreLink"></breadcrumb>
+          <breadcrumb :linkobject="parentExploreLink" :text="$tr('back')"></breadcrumb>
```

Comment:
This still shows a 'back' arrow though - so now we have a back arrow with 'up' next to it. CONFUSING!